push.sh

```diff
@@ -66,9 +66,16 @@ fi
 
 indent() { sed 's/^/  /'; }
 
+declare HELM3_VERSION="$(helm --client --short version | grep "v3\.")"
```

Comment:
This needs to be `helm version --client`...

Reply:
I've just checked on both Helm 2 & Helm 3 - options positioning does not affect the output. It's the same. But changed it anyway. Please check when you have a time. Thank you.

Reply:
At least with Helm 2.9.1, it did not work:
```$ helm --client version
Error: unknown flag: --client
$ helm version --client
Client: &version.Version{SemVer:"v2.9.1", GitCommit:"20adb27c7c5868466912eebdf6664e7390ebe710", GitTreeState:"clean"}
```